waspc/src/Wasp/Message.hs

```diff
@@ -0,0 +1,5 @@
+module Wasp.Message (Message (..), SendMessage) where
+
+data Message = StartMessage String | SuccessMessage String | WarningMessage String | ErrorMessage String
```

Comment:
Let's comment about the ErrorMessage that it shouldn't be handled as an error, it is just a message. Maybe rename it to FailureMessage.